### ISSUE TITLE
SocketRocket: In debug mode. Allowing connection to any root cert

### ISSUE THREAD

**Issue by likeaj6** [GetStream/stream-chat-react-native#608]
**Describe the bug**

Getting this error message: `SocketRocket: In debug mode. Allowing connection to any root cert. `

Seems to be an underlying socket/connectivity issue, but unclear what's causing it/how to resolve it when it occurs. It happens relatively sporadically, but the messages in the channel all disappear when it occurs and so it's pretty breaking for our use case. Sending a message seems to help reconnect! Is there a way to better handle it when it happens so messages don't just disappear completely?

**Dev environment info (please complete/provide the following information):**

- "stream-chat": "3.7.0",
- "stream-chat-react-native": "3.2.0",
- "react-native": "0.64.0"
- Device/Emulator: iOS Simulator and iPhone 10, iOS 14.5

**Comment by likeaj6** [comment]
Wanted to follow up on this - I haven't been able to pinpoint whether the error message is the cause of the issue with messages disappearing and empty state being shown, but messages disappearing completely happens often enough in our solution for it to be a deal-breaker. We've tried adding a reload button to re-watch the channel, which appears to help sometimes, but is not a great solution.

We migrated from a custom-built gifted-chat solution built on top of redux, so it's a little surprising to be running into breaking connectivity issues for a dedicated chat service (especially since there is no caching/offline support).
